.changeset/new-owls-perform.md

```diff
@@ -0,0 +1,5 @@
+---
+"react-magma-dom": patch
```

Comment:
this is in the dropzone package not react-magma-dom

Reply:
🤦‍♀️  fixed